app/components/StyledCard/index.js

```diff
@@ -106,6 +108,11 @@ function StyledCard(props) {
     }
   }
 
+  function resolveLink(trackId) {
+    const path = routeConstants.details.route;
+    return path.replace(':id', trackId);
+  }
+
   return (
     <If condition={!isEmpty(musicData.results)} otherwise={<T data-testid="no-music-data" id="no_results" />}>
       {musicData.results.slice(0, 10).map((item, index) => {
```

Comment:
just use `For` instead. There's no problem with this, its just that the For reads better